docs/source/tutorials/basics/basics_backend.rst

```diff
@@ -63,7 +63,9 @@ with a SageMaker estimator, which executes the training script. In our example,
 we use the ``PyTorch`` SageMaker framework as a pre-built container for the
 dependencies our training scripts requires. However, any other type of
 `SageMaker estimator <https://sagemaker.readthedocs.io/en/stable/api/training/estimators.html>`_
-can be used here just as well.
+can be used here just as well. Finally, if you include any of the metrics reported
+by your training script in ``metrics_names``, their values are visualized in the
+dashboard for the SageMaker training job.
```

Comment:
How does the SageMaker side know to graph these logs?

Reply:
What do you mean? They are passed as `metric_names` to the estimator, and they are output by the `Reporter`. Then, SageMaker greps the logs.

We also grep the logs on our end (the SM backend does), so to receive new values. David did it this way to stay compatible with SageMaker.

Reply:
Got it, thanks!